source/geod24/LocalRest.d

```diff
@@ -1041,8 +1060,10 @@ unittest
 
         override size_t fooCount() { return this.foo_count; }
         override size_t fooIntCount() { return this.foo_int_count; }
+        override size_t barCount() { return this.bar_count; }
         override void foo () { ++this.foo_count; }
         override void foo (int) { ++this.foo_int_count; }
+        override void bar (int) { ++this.bar_count; }  // not in any overload set
```

Comment:
```suggestion
        override void bar (int) { ++this.bar_count; }  // not in any overload set
        // This one is part of the overload set of the node, but not of the API
        // It can't be accessed via API and can't be filtered out
        void bar(string) { assert(0); }
```